### PR TITLE
Proposed wording for issue 2158

### PR DESCRIPTION
**Pull request by AlisdairM**
Issue resolution proposed by Michael Wong and Alisdair Meredith, following some guidance by Chicago review.

### CODE REVIEW

**Comment by AlisdairM** [comment]
Doh, note that the branch name is entirely inappropriate, this should hopefully be a Tentatively Ready resolution.

File: xml/issue2158.xml [hunk 1]
```diff
@@ -47,9 +47,113 @@ Maybe it would be useful to introduce a new name for such conditional requiremen
...
+  </td>
+  <td>linear</td>
+</tr>
+<tr>
+  <td><tt>X u(a)</tt><br/><tt>X u = a;</tt></td>
+  <td></td>
+  <td></td>
+  <td>
+    <i>Requires:</i> <tt>T</tt> is <tt>CopyInsertable</tt> into
+    <tt>X</tt> (see below)<del>.</del><ins>, otherwise this expression shall be ill-formed.</ins><br/>
```
Comment (timsong-cpp) [inline xml/issue2158.xml]: This isn't an expression.

Comment (timsong-cpp) [inline xml/issue2158.xml]: `X u(a)` and `X u = a;` aren't expressions.

Comment (timsong-cpp) [inline xml/issue2158.xml]: `CopyInsertable` has a semantic component that can't be checked, and even `allocator_traits<A>::construct(m, p, v)` isn't checkable in the general case.

**Pull request merged**